HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -260,62 +207,54 @@ private String writeFile() {
 	private class HVConnect extends AsyncTask<Void, Void, Void> {
 		
 		private ProgressDialog progressDialog;
-        
-        public HVConnect() {
-            progressDialog = ProgressDialog.show(
-                    MainActivity.this,
-                    "",
-                    "Please wait...", 
-                    true);
-        }
+
+		public HVConnect() {
+			progressDialog = ProgressDialog.show(MainActivity.this, "", "Please wait...", true);
+		}
 
 		@Override
 		protected Void doInBackground(Void... arg0) {
 			
-			HealthVaultSettings settings = service.getSettings();
+			HealthVaultSettings settings = mService.getSettings();
 			settings.setMasterAppId("c6ba979f-c342-4408-a2bc-0dfb43b2bf8d");
```

Comment:
why setting it here istead of reading the file?

---
In reply to: [118800917](https://github.com/Microsoft/healthvault-java-sdk/pull/5#discussion_r118800917) [](ancestors = 118800917)